src/components/ListingDetails/styles.ts

```diff
@@ -1,15 +1,11 @@
 import styled from 'styled-components';
 import COLORS from '@/styles/colors';
-import { H1, P } from '@/styles/text';
+import { P } from '@/styles/text';
 
-export const CaseDisplay = styled.aside`
-  position: relative;
-  width: 100%;
+export const ListingDescription = styled(P)`
+  word-break: break-all;
```

Comment:
The reason for adding this style is to prevent overflow from a long text:
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/5fc7ee2a-7e8b-44e5-95b0-28df62498d0d)

But you're right - this style isn't the best as it would break any character in the word. Perhaps it may be best if we change it to `overflow-wrap: break-word`?
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/fe97fc03-8851-44a3-aa59-9b06d1ecdad7)

This should in theory only break long words that would otherwise not fit in one line and cause overflow.

Reply:
Yeah `break-word` seems like the best option here. Thanks jinkang!